envs/.env.tugboat

```diff
@@ -18,3 +18,7 @@ NEXT_PUBLIC_GOOGLE_TAG_MANAGER_PREVIEW=env-423
 # For Drupal preview
 DRUPAL_PREVIEW_SECRET=secret
 # DRUPAL_CLIENT_ID & DRUPAL_CLIENT_SECRET are made available through Tugboat UI settings.
+
+# Configure the localhost port for the Next.js server. This is used for Preview on CMS Tugboats
+# next-build tugboat serve static files, no port necessary.
```

Comment:
I don't understand this line `next-build tugboat serve static files, no port necessary.` but the first comment makes sense.